bors.toml

```diff
@@ -0,0 +1,4 @@
+# Gate on both Travis CI and Cirrus-ci
+status = ["continuous-integration/appveyor/branch",
+	  "continuous-integration/travis-ci/push",
+	  "stable x86_64-unknown-freebsd"]
```

Comment:
I don't think the homu/bors combo we have recognizes this file?

Reply:
I think [this may need to be modified](https://github.com/rust-lang/rust-central-station/blob/d28d4fd292bc3a7db495ba93b4065858c9428230/homu.toml.template#L244-L265) though to gate on Cirrus CI

Reply:
The docs say that it's required, and don't mention any default values.  As is, I suspect that bors would allow you to merge a PR with a failing build status.

Reply:
Does libc have its own fork of bors or something?

Reply:
No it uses the same as rust-lang/rust, which doesn't need this file